autogpt_platform/backend/schema.prisma

```diff
@@ -553,7 +555,7 @@ model StoreListingVersion {
   StoreListingReview StoreListingReview[]
 
   @@unique([agentId, agentVersion])
-  @@index([agentId, agentVersion, isApproved])
+  @@index([agentId, agentVersion, isDeleted])
```

Comment:
AgentId and agent version is already unique

Adding another index on this will be useless and slowing down the write